components/content/robocomp/robocomp-teams.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <lazy-client-only>
+    <span v-if="status === 'pending'">Ładowanie...</span>
+
+    <fwb-accordion v-else-if="status === 'success' && teamsData && !(teamsData instanceof H3Error)" always-open>
+      <fwb-accordion-panel v-for="(team, index) in teamsData.data.teams" :key="index">
+        <fwb-accordion-header>{{ team.name }}</fwb-accordion-header>
+        <fwb-accordion-content>
+          <div>
+            <h3 class="dark:text-gray-400">Robots:</h3>
+            <ul>
+              <li class="dark:text-gray-400"
+                v-for="(robot, index) in team.robots.split(';').map(
+                  (robot) =>
+                    ({
+                      name: robot.split('<').slice(0, 2).join(' - ').split('>').slice(0,1).join()
+                    }) as Robot
+                )"
+              >
+                {{ robot.name }}
+              </li>
+            </ul>
+          </div>
+        </fwb-accordion-content>
+      </fwb-accordion-panel>
+    </fwb-accordion>
+  </lazy-client-only>
+</template>
+
+<script lang="ts" setup>
```

Comment:
script setup lang

Reply:
It was actually automatically generated by this extension https://marketplace.visualstudio.com/items?itemName=Nuxtr.nuxtr-vscode#personalized-vue-file-templates

does the order really matter?

Reply:
I'd like to have the same order everywhere ;)

Reply:
ye anyway it is changed